### PR TITLE
programs: write yaml description files for the rest tests

### PR DESCRIPTION
**Pull request by jiaqizhao122**

### CODE REVIEW

File: programs/hwsim/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+description: >
+ This is a wireless medium simulation tool for Linux, based on the netlink API implemented in the mac80211_hwsim kernel driver. Unlike the default in-kernel forwarding mode of mac80211_hwsim, wmediumd allows simulating frame loss and delay.
+
+homepage: https://github.com/bcopeland/wmediumd.git
+
+parameters:
+ test：
+ 
+results:
```
Comment (rli9) [inline programs/hwsim/meta.yaml:9]: no need to list results for hwsim

File: programs/igt/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,345 @@
...
+results. Therefore, IGT GPU Tools includes low-level tools and tests
+specifically for development and testing of the DRM Drivers.
+
+homepage: https://gitlab.freedesktop.org/drm/igt-gpu-tools
+
+parameters:
+ group:
+ test:
+
+results:
```
Comment (rli9) [inline programs/igt/meta.yaml:16]: no need to list results

File: programs/igt/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,345 @@
+description: >
```
Comment (rli9) [inline programs/igt/meta.yaml:1]: Pls add "short_description: >" for all applicable tests.

File: programs/analyze-suspend/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+description: >
```
Comment (rli9) [inline programs/analyze-suspend/meta.yaml:1]: need short_description: >, similar comments for other tests.

File: programs/avocado/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+ a set of tools and libraries to help with automated testing.
+ One can call it a test framework with benefits. Native tests are written in Python
+ and they follow the unittest pattern, but any executable can serve as a test.
+
+homepage: https://github.com/avocado-framework
+
+parameters:
+ test_group:
+
+results:
```
Comment (rli9) [inline programs/avocado/meta.yaml:11]: no need to have results for this functional test

File: programs/blktests/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+ It was originally written by Omar Sandoval and announced in 2017.
+
+homepage: https://github.com/osandov/blktests
+
+parameters:
+ test:
+ nvme_trtype:
+ use_siw:
+
+results:
```
Comment (rli9) [inline programs/blktests/meta.yaml:13]: no need to have results for this functional test

File: programs/hwsim/meta.yaml (hunk 1 above)
Comment (jiaqizhao122) [reply programs/hwsim/meta.yaml:9]: ok

File: programs/igt/meta.yaml (hunk 2 above)
Comment (jiaqizhao122) [reply programs/igt/meta.yaml:16]: ok

File: programs/avocado/meta.yaml (hunk 5 above)
Comment (jiaqizhao122) [reply programs/avocado/meta.yaml:11]: ok

File: programs/blktests/meta.yaml (hunk 6 above)
Comment (jiaqizhao122) [reply programs/blktests/meta.yaml:13]: ok

File: programs/analyze-suspend/meta.yaml (hunk 4 above)
Comment (jiaqizhao122) [reply programs/analyze-suspend/meta.yaml:1]: ok

File: programs/igt/meta.yaml (hunk 3 above)
Comment (jiaqizhao122) [reply programs/igt/meta.yaml:1]: ok

File: programs/blktests/meta.yaml (hunk 6 above)
Comment (rli9) [reply programs/blktests/meta.yaml:15]: Is this resolved? The results are still there

File: programs/hwsim/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,10 @@
+short_description: >
```
Comment (rli9) [inline programs/hwsim/meta.yaml:1]: pls use This is a wireless medium simulation tool for Linux, based on the netlink API implemented in the mac80211_hwsim kernel driver.

File: programs/igt/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,17 @@
+short_description: >
```
Comment (rli9) [inline programs/igt/meta.yaml:1]: the content is missing

File: programs/f2fs-tools/meta.yaml [hunk 9]
```diff
@@ -0,0 +1,10 @@
+short_description: >
```
Comment (rli9) [inline programs/f2fs-tools/meta.yaml:1]: same as description

File: programs/filebench/meta.yaml [hunk 10]
```diff
@@ -0,0 +1,18 @@
+short_description: >
```
Comment (rli9) [inline programs/filebench/meta.yaml:1]: use "Filebench is a file system and storage benchmark that can generate a large
variety of workloads."

File: programs/leaking-addresses/meta.yaml [hunk 11]
```diff
@@ -0,0 +1,143 @@
+short_description: >
+
+description: >
+
+homepage: https://raw.githubusercontent.com/torvalds/linux/master/scripts/leaking_addresses.pl
+
+parameters:
+
+results:
```
Comment (rli9) [inline programs/leaking-addresses/meta.yaml:9]: can remove the results

File: programs/makepkg/meta.yaml [hunk 12]
```diff
@@ -0,0 +1,12 @@
+short_description: >
+ fakeroot can be built to either use SYSV IPC or TCP for message passing.
```
Comment (rli9) [inline programs/makepkg/meta.yaml:2]: thanks, no need to add short description, ok to leave empty

File: programs/makepkg/meta.yaml [hunk 13]
```diff
@@ -0,0 +1,12 @@
...
+
+description: >
+
+homepage: https://github.com/mackyle/fakeroot
```
Comment (rli9) [inline programs/makepkg/meta.yaml:6]: leave empty is ok

File: programs/mcperf/meta.yaml [hunk 14]
```diff
@@ -0,0 +1,17 @@
+short_description: >
+ twemperf (pronounced "two-em-perf"), aka mcperf is a tool for measuring memcached server performance. mcperf is like httperf, but for memcached protocol. It speaks memcached ASCII protocol and is capable of generating connections and requests at a high rate.
+
+description: >
```
Comment (rli9) [inline programs/mcperf/meta.yaml:4]: use "mcperf is a tool for measuring memcached server performance. mcperf is like httperf, but for memcached protocol. It speaks memcached ASCII protocol and is capable of generating connections and requests at a high rate."

File: programs/kunit/meta.yaml [hunk 15]
```diff
@@ -0,0 +1,91 @@
...
+
+description: >
+
+homepage:
+
+parameters:
+ group:
+
+results:
+ kunit.kunit-try-catch-test.kunit_test_try_catch_successful_try_no_catch.pass:
```
Comment (rli9) [inline programs/kunit/meta.yaml:11]: remove result

File: programs/kbuild/meta.yaml [hunk 16]
```diff
@@ -0,0 +1,19 @@
...
+homepage: https://www.kernel.org/pub/linux/kernel
+
+parameters:
+ nr_task:
+ runtime:
+ build_kconfig:
+ target:
+
+results:
+ kbuildbuildtime_per_iteration:
```
Comment (rli9) [inline programs/kbuild/meta.yaml:14]: this stat is named wrongly

File: programs/blktests/meta.yaml (hunk 6 above)
Comment (jiaqizhao122) [reply programs/blktests/meta.yaml:15]: ok

File: programs/hwsim/meta.yaml (hunk 7 above)
Comment (jiaqizhao122) [reply programs/hwsim/meta.yaml:1]: ok

File: programs/gcc/meta.yaml [hunk 17]
```diff
@@ -0,0 +1,10 @@
+short_description: >
```
[the remark this answers is not in the document]
Comment (jiaqizhao122) [reply programs/gcc/meta.yaml:1]: ok

File: programs/igt/meta.yaml (hunk 8 above)
Comment (jiaqizhao122) [reply programs/igt/meta.yaml:1]: ok

File: programs/f2fs-tools/meta.yaml (hunk 9 above)
Comment (jiaqizhao122) [reply programs/f2fs-tools/meta.yaml:1]: ok

File: programs/energy/meta.yaml [hunk 18]
```diff
@@ -0,0 +1,10 @@
+short_description: >
```
[the remark this answers is not in the document]
Comment (jiaqizhao122) [reply programs/energy/meta.yaml:1]: ok

File: programs/filebench/meta.yaml (hunk 10 above)
Comment (jiaqizhao122) [reply programs/filebench/meta.yaml:1]: ok

File: programs/leaking-addresses/meta.yaml (hunk 11 above)
Comment (jiaqizhao122) [reply programs/leaking-addresses/meta.yaml:9]: ok

File: programs/makepkg/meta.yaml (hunk 12 above)
Comment (jiaqizhao122) [reply programs/makepkg/meta.yaml:2]: ok

File: programs/mcperf/meta.yaml [hunk 19]
```diff
@@ -0,0 +1,17 @@
+short_description: >
+ twemperf (pronounced "two-em-perf"), aka mcperf is a tool for measuring memcached server performance. mcperf is like httperf, but for memcached protocol. It speaks memcached ASCII protocol and is capable of generating connections and requests at a high rate.
```
[the remark this answers is not in the document]
Comment (jiaqizhao122) [reply programs/mcperf/meta.yaml:2]: ok

File: programs/mcperf/meta.yaml (hunk 14 above)
Comment (jiaqizhao122) [reply programs/mcperf/meta.yaml:4]: ok

File: programs/kunit/meta.yaml (hunk 15 above)
Comment (jiaqizhao122) [reply programs/kunit/meta.yaml:11]: ok

File: programs/kbuild/meta.yaml (hunk 16 above)
Comment (jiaqizhao122) [reply programs/kbuild/meta.yaml]: ok

**Comment by rli9** [comment]
thanks for the patch

**Pull request merged**